source/includes/fact-natural-language-querying-maxTimems.txt

```diff
@@ -0,0 +1,5 @@
+When using natural language querying, you may need to modify the 
+query's maximum execution time by adjusting the :guilabel:`MAX TIME MS` 
+query bar option. This option safeguards your cluster from 
+excessively long query times. For details, see 
```

Comment:
[simplification suggestion]

This option prevents long-running queries on your cluster.

Reply:
Updated.